roles/ceph-docker-common/tasks/fetch_configs.yml

```diff
@@ -9,6 +9,7 @@
       - /var/lib/ceph/bootstrap-osd/{{ cluster }}.keyring
       - /var/lib/ceph/bootstrap-rgw/{{ cluster }}.keyring
       - /var/lib/ceph/bootstrap-mds/{{ cluster }}.keyring
+      - /var/lib/ceph/bootstrap-rbd-mirror/{{ cluster }}.keyring
```

Comment:
Not yet -- I am working right now to handle pre-Luminous where this bootstrap key will never by auto-created and >= Luminous where it will be available